tests/Transports.Subscriptions.Abstractions.Tests/ProtocolHandlerFacts.cs

```diff
@@ -231,5 +233,12 @@ public async Task Receive_unknown()
             Assert.Contains(_transportWriter.WrittenMessages,
                 message => message.Type == MessageType.GQL_CONNECTION_ERROR);
         }
+
+        private JObject FromObject(object value)
+        {
+            var serializer = new GraphQLSerializer();
+            var data = serializer.Serialize(value);
+            return serializer.Deserialize<JObject>(data);
+        }
```

Comment:
Because `GraphQLRequest` is not tagged with the Newtonsoft-specific attributes to indicate the proper key name, a serializer must be used.  (We do not provide a method to serialize to a `JObject`.)